lib/streaming-pipeline.js

```diff
@@ -118,6 +127,10 @@ module.exports = class StreamingPipeline extends Writable {
         this._functionInputs.forEach((fi) => fi.emit('end'));
     }
 
+    _errorInputs(err) {
+        this._functionInputs.forEach((fi) => fi.emit('error', err));
```

Comment:
I think this is backwards. What I would expect to see is something that listen to each input stream passed into the function for unmarshaling errors, since we can't assume the user function will handle that correctly. If any of the input streams err, pass that error back via the grpc stream.

Reply:
I'm following the [spec](https://github.com/projectriff/invoker-specification/blob/master/streaming.md#stream-completion-and-error-conditions) here:
```
When an invoker receives the error stream completion signal on its input stream, it MUST propagate that error termination to all of the function input streams.
```